exchanges/stream/websocket.go

```diff
@@ -577,7 +578,6 @@ func (w *Websocket) trafficMonitor() {
 					}
 				}
 				w.setTrafficMonitorRunning(false)
```

Comment:
I think it might better to move this above the Done() call just so in the unlikely event when a traffic monitor starts up again and this isn't set to false. It might hang differently. Let me know what you think?

Other than that looks good! Thanks for checking this out.

Reply:
Makes sense. I’ve updated it.